site/src/api/typesGenerated.ts

```diff
@@ -1640,19 +1640,14 @@ export interface Workspace {
   readonly owner_name: string;
   readonly owner_avatar_url: string;
   readonly organization_id: string;
-  readonly template_id: string;
-  readonly template_name: string;
-  readonly template_display_name: string;
-  readonly template_icon: string;
-  readonly template_allow_user_cancel_workspace_jobs: boolean;
-  readonly template_active_version_id: string;
-  readonly template_require_active_version: boolean;
```

Comment:
I always wonder why we just return the template instead of selecting attribute by attribute.

Reply:
I bet it just grew over time.